### PR TITLE
chore(docs): add anchors to environment variables

### PR DESCRIPTION
**Pull request by majorgreys**
## Description

Add `id` to table rows that can be used to refer to environment variables in the configuration.

E.g., https://788960-61572326-gh.circle-artifacts.com/0/tmp/docs/configuration.html#dd-profiling-capture-pct

## Checklist
- [ ] Added to the correct milestone.
- [ ] Tests provided or description of manual testing performed is included in the code or PR.
- [ ] Library documentation is updated.
- [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).

### CODE REVIEW

**Review by Kyle-Verhoog — DISMISSED** [review]

File: docs/configuration.rst [hunk 1]
```diff
@@ -15,130 +15,186 @@ below:
...
      - 60
      - The interval in seconds to wait before flushing out recorded events.
+
+       .. _dd_profiling_ignore_profiler:
    * - ``DD_PROFILING_IGNORE_PROFILER``
      - Boolean
      - False
      - **Deprecated**: whether to ignore the profiler in the generated data.
+
+       .. _dd_profiling_tags:
```
Comment (Yun-Kim) [inline docs/configuration.rst:197]: 👍

File: docs/configuration.rst [hunk 2]
```diff
@@ -15,130 +15,186 @@ below:
...
        allocation). Greater values reduce the program execution speed. Must be
        greater than 0 lesser or equal to 100.
+
+       .. _dd_profiling_upload_interval:
    * - ``DD_PROFILING_UPLOAD_INTERVAL``
      - Float
      - 60
      - The interval in seconds to wait before flushing out recorded events.
+
+       .. _dd_profiling_ignore_profiler:
```
Comment (Yun-Kim) [inline docs/configuration.rst:191]: 👍

File: docs/configuration.rst [hunk 3]
```diff
@@ -15,130 +15,186 @@ below:
...
+
+       .. _dd_profiling_capture_pct:
    * - ``DD_PROFILING_CAPTURE_PCT``
      - Float
      - 2
      - The percentage of events that should be captured (e.g. memory
        allocation). Greater values reduce the program execution speed. Must be
        greater than 0 lesser or equal to 100.
+
+       .. _dd_profiling_upload_interval:
```
Comment (Yun-Kim) [inline docs/configuration.rst:185]: 👍

File: docs/configuration.rst [hunk 4]
```diff
@@ -15,130 +15,186 @@ below:
...
      - 64
      - The maximum number of frames to capture in stack execution tracing.
+
+       .. _dd_profiling_heap_enabled:
    * - ``DD_PROFILING_HEAP_ENABLED``
      - Boolean
      - False
      - Whether to enable the heap memory profiler.
+
+       .. _dd_profiling_capture_pct:
```
Comment (Yun-Kim) [inline docs/configuration.rst:177]: 👍

File: docs/configuration.rst [hunk 5]
```diff
@@ -15,130 +15,186 @@ below:
...
      - The percentage of maximum time the stack profiler can use when computing
        statistics. Must be greater than 0 and lesser or equal to 100.
+
+       .. _dd_profiling_max_frames:
    * - ``DD_PROFILING_MAX_FRAMES``
      - Integer
      - 64
      - The maximum number of frames to capture in stack execution tracing.
+
+       .. _dd_profiling_heap_enabled:
```
Comment (Yun-Kim) [inline docs/configuration.rst:171]: 👍

File: docs/configuration.rst [hunk 6]
```diff
@@ -15,130 +15,186 @@ below:
...
        reply.
+
+       .. _dd_profiling_max_time_usage_pct:
    * - ``DD_PROFILING_MAX_TIME_USAGE_PCT``
      - Float
      - 1
      - The percentage of maximum time the stack profiler can use when computing
        statistics. Must be greater than 0 and lesser or equal to 100.
+
+       .. _dd_profiling_max_frames:
```
Comment (Yun-Kim) [inline docs/configuration.rst:165]: 👍

File: docs/configuration.rst [hunk 7]
```diff
@@ -15,130 +15,186 @@ below:
...
      - Enable Datadog profiling when using ``ddtrace-run``.
+
+       .. _dd_profiling_api_timeout:
    * - ``DD_PROFILING_API_TIMEOUT``
      - Float
      - 10
      - The timeout in seconds before dropping events if the HTTP API does not
        reply.
+
+       .. _dd_profiling_max_time_usage_pct:
```
Comment (Yun-Kim) [inline docs/configuration.rst:158]: 👍

File: docs/configuration.rst [hunk 8]
```diff
@@ -15,130 +15,186 @@ below:
...
      - 1.0
      - A float, f, 0.0 <= f <= 1.0. f*100% of traces will be sampled.
+
+       .. _dd_profiling_enabled:
    * - ``DD_PROFILING_ENABLED``
      - Boolean
      - False
      - Enable Datadog profiling when using ``ddtrace-run``.
+
+       .. _dd_profiling_api_timeout:
```
Comment (Yun-Kim) [inline docs/configuration.rst:151]: 👍

File: docs/configuration.rst [hunk 9]
```diff
@@ -15,130 +15,186 @@ below:
...
      - False
      - Enable or disable start up diagnostic logging.
+
+       .. _dd_trace_sample_rate:
    * - ``DD_TRACE_SAMPLE_RATE``
      - Float
      - 1.0
      - A float, f, 0.0 <= f <= 1.0. f*100% of traces will be sampled.
+
+       .. _dd_profiling_enabled:
```
Comment (Yun-Kim) [inline docs/configuration.rst:145]: 👍

File: docs/configuration.rst [hunk 10]
```diff
@@ -15,130 +15,186 @@ below:
...
      - 1.0
      - The time between each flush of traces to the trace agent.
+
+       .. _dd_trace_startup_logs:
    * - ``DD_TRACE_STARTUP_LOGS``
      - Boolean
      - False
      - Enable or disable start up diagnostic logging.
+
+       .. _dd_trace_sample_rate:
```
Comment (Yun-Kim) [inline docs/configuration.rst:139]: 👍

File: docs/configuration.rst [hunk 11]
```diff
@@ -15,130 +15,186 @@ below:
...
      - 8000000
      - The max size in bytes of each payload sent to the trace agent. If max payload size is less than buffer size, multiple payloads will be sent to the trace agent.
+
+       .. _dd-trace-writer-interval-seconds:
    * - ``DD_TRACE_WRITER_INTERVAL_SECONDS``
      - Float
      - 1.0
      - The time between each flush of traces to the trace agent.
+
+       .. _dd_trace_startup_logs:
```
Comment (Yun-Kim) [inline docs/configuration.rst:133]: 👍

**Review by Yun-Kim — COMMENTED** [review]
👍  --> underscore ids

**Review by Yun-Kim — APPROVED** [review]

**Review by Kyle-Verhoog — APPROVED** [review]

File: docs/configuration.rst [hunk 12]
```diff
@@ -15,130 +15,186 @@ below:
...
      - 60
      - The interval in seconds to wait before flushing out recorded events.
+
+       .. _dd-profiling-ignore-profiler:
    * - ``DD_PROFILING_IGNORE_PROFILER``
      - Boolean
      - False
      - **Deprecated**: whether to ignore the profiler in the generated data.
+
+       .. _dd-profiling-tags:
```
Comment (brettlangdon) [inline docs/configuration.rst:197]: 👍🏻

File: docs/configuration.rst [hunk 13]
```diff
@@ -15,130 +15,186 @@ below:
...
+
+       .. _dd-profiling-capture-pct:
    * - ``DD_PROFILING_CAPTURE_PCT``
      - Float
      - 2
      - The percentage of events that should be captured (e.g. memory
        allocation). Greater values reduce the program execution speed. Must be
        greater than 0 lesser or equal to 100.
+
+       .. _dd-profiling-upload-interval:
```
Comment (brettlangdon) [inline docs/configuration.rst:185]: 👍🏻

File: docs/configuration.rst [hunk 14]
```diff
@@ -15,130 +15,186 @@ below:
...
      - 64
      - The maximum number of frames to capture in stack execution tracing.
+
+       .. _dd-profiling-heap-enabled:
    * - ``DD_PROFILING_HEAP_ENABLED``
      - Boolean
      - False
      - Whether to enable the heap memory profiler.
+
+       .. _dd-profiling-capture-pct:
```
Comment (brettlangdon) [inline docs/configuration.rst:177]: 👍🏻

File: docs/configuration.rst [hunk 15]
```diff
@@ -15,130 +15,186 @@ below:
...
      - The percentage of maximum time the stack profiler can use when computing
        statistics. Must be greater than 0 and lesser or equal to 100.
+
+       .. _dd-profiling-max-frames:
    * - ``DD_PROFILING_MAX_FRAMES``
      - Integer
      - 64
      - The maximum number of frames to capture in stack execution tracing.
+
+       .. _dd-profiling-heap-enabled:
```
Comment (brettlangdon) [inline docs/configuration.rst:171]: 👍🏻

File: docs/configuration.rst [hunk 16]
```diff
@@ -15,130 +15,186 @@ below:
...
        reply.
+
+       .. _dd-profiling-max-time-usage-pct:
    * - ``DD_PROFILING_MAX_TIME_USAGE_PCT``
      - Float
      - 1
      - The percentage of maximum time the stack profiler can use when computing
        statistics. Must be greater than 0 and lesser or equal to 100.
+
+       .. _dd-profiling-max-frames:
```
Comment (brettlangdon) [inline docs/configuration.rst:165]: 👍🏻

File: docs/configuration.rst [hunk 17]
```diff
@@ -15,130 +15,186 @@ below:
...
      - Enable Datadog profiling when using ``ddtrace-run``.
+
+       .. _dd-profiling-api-timeout:
    * - ``DD_PROFILING_API_TIMEOUT``
      - Float
      - 10
      - The timeout in seconds before dropping events if the HTTP API does not
        reply.
+
+       .. _dd-profiling-max-time-usage-pct:
```
Comment (brettlangdon) [inline docs/configuration.rst:158]: 👍🏻

File: docs/configuration.rst [hunk 18]
```diff
@@ -15,130 +15,186 @@ below:
...
      - 1.0
      - A float, f, 0.0 <= f <= 1.0. f*100% of traces will be sampled.
+
+       .. _dd-profiling-enabled:
    * - ``DD_PROFILING_ENABLED``
      - Boolean
      - False
      - Enable Datadog profiling when using ``ddtrace-run``.
+
+       .. _dd-profiling-api-timeout:
```
Comment (brettlangdon) [inline docs/configuration.rst:151]: 👍🏻

File: docs/configuration.rst [hunk 19]
```diff
@@ -15,130 +15,186 @@ below:
...
      - False
      - Enable or disable start up diagnostic logging.
+
+       .. _dd-trace-sample-rate:
    * - ``DD_TRACE_SAMPLE_RATE``
      - Float
      - 1.0
      - A float, f, 0.0 <= f <= 1.0. f*100% of traces will be sampled.
+
+       .. _dd-profiling-enabled:
```
Comment (brettlangdon) [inline docs/configuration.rst:145]: 👍🏻

File: docs/configuration.rst [hunk 20]
```diff
@@ -15,130 +15,186 @@ below:
...
      - 1.0
      - The time between each flush of traces to the trace agent.
+
+       .. _dd-trace-startup-logs:
    * - ``DD_TRACE_STARTUP_LOGS``
      - Boolean
      - False
      - Enable or disable start up diagnostic logging.
+
+       .. _dd-trace-sample-rate:
```
Comment (brettlangdon) [inline docs/configuration.rst:139]: 👍🏻

File: docs/configuration.rst [hunk 21]
```diff
@@ -15,130 +15,186 @@ below:
...
      - 8000000
      - The max size in bytes of each payload sent to the trace agent. If max payload size is less than buffer size, multiple payloads will be sent to the trace agent.
+
+       .. _dd-trace-writer-interval-seconds:
    * - ``DD_TRACE_WRITER_INTERVAL_SECONDS``
      - Float
      - 1.0
      - The time between each flush of traces to the trace agent.
+
+       .. _dd-trace-startup-logs:
```
Comment (brettlangdon) [inline docs/configuration.rst:133]: 👍🏻

File: docs/configuration.rst [hunk 22]
```diff
@@ -15,130 +15,186 @@ below:
...
      - 8000000
      - The max size in bytes of traces to buffer between flushes to the agent.
+
+       .. _dd-trace-writer-max-payload-size-bytes:
    * - ``DD_TRACE_WRITER_MAX_PAYLOAD_SIZE_BYTES``
      - Int
      - 8000000
      - The max size in bytes of each payload sent to the trace agent. If max payload size is less than buffer size, multiple payloads will be sent to the trace agent.
+
+       .. _dd-trace-writer-interval-seconds:
```
Comment (brettlangdon) [inline docs/configuration.rst:127]: 👍🏻

File: docs/configuration.rst [hunk 23]
```diff
@@ -15,130 +15,186 @@ below:
...
      - 2.0
      - The timeout in float to use to connect to the Datadog agent.
+
+       .. _dd-trace-writer-buffer-size-bytes:
    * - ``DD_TRACE_WRITER_BUFFER_SIZE_BYTES``
      - Int
      - 8000000
      - The max size in bytes of traces to buffer between flushes to the agent.
+
+       .. _dd-trace-writer-max-payload-size-bytes:
```
Comment (brettlangdon) [inline docs/configuration.rst:121]: 👍🏻

File: docs/configuration.rst [hunk 24]
```diff
@@ -15,130 +15,186 @@ below:
...
        ``http://`` to connect using HTTP or with ``unix://`` to use a Unix
        Domain Socket.
+
+       .. _dd-trace-agent-timeout-seconds:
    * - ``DD_TRACE_AGENT_TIMEOUT_SECONDS``
      - Float
      - 2.0
      - The timeout in float to use to connect to the Datadog agent.
+
+       .. _dd-trace-writer-buffer-size-bytes:
```
Comment (brettlangdon) [inline docs/configuration.rst:115]: 👍🏻

File: docs/configuration.rst [hunk 25]
```diff
@@ -15,130 +15,186 @@ below:
...
+
+       .. _dd-trace-agent-url:
    * - ``DD_TRACE_AGENT_URL``
      - URL
      - ``http://localhost:8126``
      - The URL to use to connect the Datadog agent. The url can starts with
        ``http://`` to connect using HTTP or with ``unix://`` to use a Unix
        Domain Socket.
+
+       .. _dd-trace-agent-timeout-seconds:
```
Comment (brettlangdon) [inline docs/configuration.rst:109]: 👍🏻

File: docs/configuration.rst [hunk 26]
```diff
@@ -15,130 +15,186 @@ below:
...
      - True
      - Enables :ref:`Logs Injection`.
+
+       .. _dd-call-basic-config:
    * - ``DD_CALL_BASIC_CONFIG``
      - Boolean
      - True
      - Controls whether ``logging.basicConfig`` is called in ``ddtrace-run`` or when debug mode is enabled.
+
+       .. _dd-trace-agent-url:
```
Comment (brettlangdon) [inline docs/configuration.rst:101]: 👍🏻

File: docs/configuration.rst [hunk 27]
```diff
@@ -15,130 +15,186 @@ below:
...
        a list in the ``module1:boolean,module2:boolean`` format. For example,
        ``boto:true,redis:false``.
+
+       .. _dd-logs-injection:
    * - ``DD_LOGS_INJECTION``
      - Boolean
      - True
      - Enables :ref:`Logs Injection`.
+
+       .. _dd-call-basic-config:
```
Comment (brettlangdon) [inline docs/configuration.rst:95]: 👍🏻

File: docs/configuration.rst [hunk 28]
```diff
@@ -15,130 +15,186 @@ below:
...
+
+       .. _datadog-patch-modules:
    * - ``DATADOG_PATCH_MODULES``
      - String
      -
      - Override the modules patched for this execution of the program. Must be
        a list in the ``module1:boolean,module2:boolean`` format. For example,
        ``boto:true,redis:false``.
+
+       .. _dd-logs-injection:
```
Comment (brettlangdon) [inline docs/configuration.rst:89]: 👍🏻

File: docs/configuration.rst [hunk 29]
```diff
@@ -15,130 +15,186 @@ below:
...
        one does not already exist. Added in ``v0.41.0`` (formerly named ``DATADOG_TRACE_DEBUG``).
+
+       .. _dd-trace-integration-enabled:
    * - ``DD_TRACE_<INTEGRATION>_ENABLED``
      - Boolean
      - True
      - Enables <INTEGRATION> to be patched. For example, ``DD_TRACE_DJANGO_ENABLED=false`` will disable the Django
        integration from being installed. Added in ``v0.41.0``.
+
+       .. _datadog-patch-modules:
```
Comment (brettlangdon) [inline docs/configuration.rst:81]: 👍🏻

File: docs/configuration.rst [hunk 30]
```diff
@@ -15,130 +15,186 @@ below:
...
        generated. Added in ``v0.41.0`` (formerly named ``DATADOG_TRACE_ENABLED``).
+
+       .. _dd-trace-debug:
    * - ``DD_TRACE_DEBUG``
      - Boolean
      - False
      - Enables debug logging in the tracer. Setting this flag will cause the library to create a root logging handler if
        one does not already exist. Added in ``v0.41.0`` (formerly named ``DATADOG_TRACE_DEBUG``).
+
+       .. _dd-trace-integration-enabled:
```
Comment (brettlangdon) [inline docs/configuration.rst:74]: 👍🏻

File: docs/configuration.rst [hunk 31]
```diff
@@ -15,130 +15,186 @@ below:
...
        ``datadoghq.eu`` to use EU site.
+
+       .. _dd-trace-enabled:
    * - ``DD_TRACE_ENABLED``
      - Boolean
      - True
      - Enable sending of spans to the Agent. Note that instrumentation will still be installed and spans will be
        generated. Added in ``v0.41.0`` (formerly named ``DATADOG_TRACE_ENABLED``).
+
+       .. _dd-trace-debug:
```
Comment (brettlangdon) [inline docs/configuration.rst:67]: 👍🏻

File: docs/configuration.rst [hunk 32]
```diff
@@ -15,130 +15,186 @@ below:
...
        ``6c44da20``, ``2020.02.13``. Generally set along with ``DD_SERVICE``. Added in ``v0.36.0``. See `Unified Service Tagging`_ for more information.
+
+       .. _dd-site:
    * - ``DD_SITE``
      - String
      - datadoghq.com
      - Specify which site to use for uploading profiles. Set to
        ``datadoghq.eu`` to use EU site.
+
+       .. _dd-trace-enabled:
```
Comment (brettlangdon) [inline docs/configuration.rst:60]: 👍🏻

File: docs/configuration.rst [hunk 33]
```diff
@@ -15,130 +15,186 @@ below:
...
      - Set global tags to be attached to every span. Value must be either comma or space separated. e.g. ``key1:value1,key2,value2`` or ``key1:value key2:value2``. Comma separated support added in ``v0.38.0`` and space separated support added in ``v0.48.0``.
+
+       .. _dd-version:
    * - ``DD_VERSION``
      - String
      -
      - Set an application's version in traces and logs e.g. ``1.2.3``,
        ``6c44da20``, ``2020.02.13``. Generally set along with ``DD_SERVICE``. Added in ``v0.36.0``. See `Unified Service Tagging`_ for more information.
+
+       .. _dd-site:
```
Comment (brettlangdon) [inline docs/configuration.rst:53]: 👍🏻

File: docs/configuration.rst [hunk 34]
```diff
@@ -15,130 +15,186 @@ below:
...
      -
      - Define service name mappings to allow renaming services in traces, e.g. ``postgres:postgresql,defaultdb:postgresql``.
+
+       .. _dd-tags:
    * - ``DD_TAGS``
      - String
      -
      - Set global tags to be attached to every span. Value must be either comma or space separated. e.g. ``key1:value1,key2,value2`` or ``key1:value key2:value2``. Comma separated support added in ``v0.38.0`` and space separated support added in ``v0.48.0``.
+
+       .. _dd-version:
```
Comment (brettlangdon) [inline docs/configuration.rst:46]: 👍🏻

File: docs/configuration.rst [hunk 35]
```diff
@@ -15,130 +15,186 @@ below:
...
        :ref:`pyramid`, :ref:`pylons`, :ref:`tornado`, :ref:`celery`, :ref:`django` and
        :ref:`falcon`. Added in ``v0.36.0``. See `Unified Service Tagging`_ for more information.
+
+       .. _dd-service-mapping:
    * - ``DD_SERVICE_MAPPING``
      - String
      -
      - Define service name mappings to allow renaming services in traces, e.g. ``postgres:postgresql,defaultdb:postgresql``.
+
+       .. _dd-tags:
```
Comment (brettlangdon) [inline docs/configuration.rst:40]: 👍🏻

File: docs/configuration.rst [hunk 36]
```diff
@@ -15,130 +15,186 @@ below:
...
+       .. _dd-service:
    * - ``DD_SERVICE``
      - String
      - (autodetected)
      - Set the service name to be used for this application. A default is
        provided for these integrations: :ref:`bottle`, :ref:`flask`, :ref:`grpc`,
        :ref:`pyramid`, :ref:`pylons`, :ref:`tornado`, :ref:`celery`, :ref:`django` and
        :ref:`falcon`. Added in ``v0.36.0``. See `Unified Service Tagging`_ for more information.
+
+       .. _dd-service-mapping:
```
Comment (brettlangdon) [inline docs/configuration.rst:34]: 👍🏻

File: docs/configuration.rst [hunk 37]
```diff
@@ -15,130 +15,186 @@ below:
...
      - Default value
      - Description
+
+       .. _dd-env:
    * - ``DD_ENV``
      - String
      -
      - Set an application's environment e.g. ``prod``, ``pre-prod``, ``staging``. Added in ``v0.36.0``. See `Unified Service Tagging`_ for more information.
+
+       .. _dd-service:
```
Comment (brettlangdon) [inline docs/configuration.rst:25]: 👍🏻

File: docs/configuration.rst [hunk 38]
```diff
@@ -15,130 +15,186 @@ below:
      - Type
      - Default value
      - Description
+
+       .. _dd-env:
```
Comment (brettlangdon) [inline docs/configuration.rst:19]: 👍🏻

**Review by brettlangdon — APPROVED** [review]
👍🏻

**Pull request merged**